newclient/src/core/components/ui/AppTimeAgo.vue

```diff
@@ -26,7 +25,7 @@ Vue.use(VueTimeago, {
         VueTimeTicker
     }
 })
-export default class AppTimeAgo extends Vue {
+export default class AppTimeAgo extends Mixins(Vue, NumberFormatMixin) {
```

Comment:
you can remove `Vue` from:
 ` Mixins(Vue, NumberFormatMixin)` and 
` import { Vue, Component, Prop, Mixins } from 'vue-property-decorator'`
It comes within the Mixin

Reply:
Also can you add localization to the strings please. so we have proper translations when switching to a different language
<img width="502" alt="Screen Shot 2022-01-28 at 1 20 16 PM" src="https://user-images.githubusercontent.com/16910687/151623088-a6b9b6e0-32be-407f-8fae-1b803bf4ab49.png">

Reply:
ok